services/blockchain-indexer/methods/dataService/controllers/blockchainApps.js

```diff
@@ -13,85 +13,46 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-// TODO: Enable when available from SDK
-// const { HTTP } = require('lisk-service-framework');
-// const dataService = require('../../../shared/dataService');
+const { HTTP } = require('lisk-service-framework');
+const dataService = require('../../../shared/dataService');
+const config = require('../../../config');
 
-// const isMainchain = async () => {
-// 	// TODO: Implement logic
-// };
+const isMainchain = async () => {
+	// TODO: Implement logic
```

Comment:
To check if it's the mainchain, please check if the last (least significant) 3 bytes of the `chainID` are zeros.
For example: On the mainnet: `00-000000` and on the testnet, `01-000000`. The first byte determines the network.